hr_infraction/models/__init__.py

```diff
@@ -0,0 +1,7 @@
+# -*- coding: utf-8 -*-
+# © 2016 OpenSynergy Indonesia
+# @author Andhitia Rama
```

Comment:
Please remvoe author line and add Michael Telahun Makonnen <mmakonnen@gmail.com>, basically it's a part of the initial __init__ file...